### DISCUSSION TITLE
Visual Editor to add Widgets

### DISCUSSION THREAD

**Discussion by peschelnet** [Lissy93/dashy#1297] [Q&A]
I'm new to using Dashy and I cannot figure out how to add widgets or display data to the dashboard using the visual editor. I know that the yaml can be edited directly, but I presume that something like adding a widget would be an option for the visual editor as well.

For example, what is my external IP widget?

- type: public-ip

Where is this put in with the visual editor to display my public ip.

Thanks

**Comment by Watever44** [comment]
It's not in the UI directly. You can go in the UI editor configuration, wich is like editing the yaml.
I prefer editing the conf.yml directly.
